### PR TITLE
Feature/merchant transfer

### PR DESCRIPTION
**Pull request by ksatya77**
Adding support for authorizations endpoint which is used for merchant based payments

### CODE REVIEW

File: src/lib/mojaloop-requests/mojaloopRequests.js [hunk 1]
```diff
@@ -72,6 +72,23 @@ class MojaloopRequests {
...
+        const url = `authorizations/${transactionRequestId}`;
+        return this._get(url, 'authorizations');
+    }
+
+    /**
+     * Executes a PUT /authorizations request for the specified transaction request id
+     */
+    async putAuthorizations(transactionRequestId, body, destFspId) {
+        const url = `authorizations/${transactionRequestId}`;
+        return this._put(url, 'parties', body, destFspId);
```
Comment (bushjames) [inline src/lib/mojaloop-requests/mojaloopRequests.js]: @ksatya77 is 'parties' resource type correct here?

Comment (ksatya77) [reply src/lib/mojaloop-requests/mojaloopRequests.js]: thx for catching that @bushjames ...updated it now...

**Comment by ksatya77** [comment]
> @ksatya77 why the change to the existing methods for authorisations?

Did not understand this comment? Can you please elaborate?

**Comment by bushjames** [comment]
@ksatya77 there was existing methods to handle authorisations, wondering what this change was for.

**Comment by ksatya77** [comment]
> @ksatya77 there was existing methods to handle authorisations, wondering what this change was for.

@bushjames you are right...Did not even notice that. Let me see if I can reject this PR.

**Comment by bushjames** [comment]
@ksatya77 is this a stale PR now? can it be closed?

**Comment by elnyry-sam-k** [comment]
Looks stale - closing this for now. please re-open with necessary changes when this can be revisited ( authorizations should be supported now in any case)

**Pull request closed without merging**